database/README.md

```diff
@@ -187,6 +237,51 @@ WHERE datasettkode='KOM'
 AND id LIKE NUMMER
 $$ LANGUAGE SQL IMMUTABLE;
 
+CREATE OR REPLACE FUNCTION public.arealstatistikk(kartlag text, wkt text)
+ RETURNS json
+AS $$
+SELECT json_object_agg(ds.datasettkode,
+   coalesce((
+     SELECT json_agg(json_build_object('kode', id, 'navn', navn,
+      'km2', km2)::jsonb)
+ FROM (
+     SELECT j.id,  j.navn, SUM(ST_Area(ST_Transform(ST_Intersection(geom, ST_GeomFromEWKT(wkt)),25833))/1000000) as km2
+     FROM kart as j
+     WHERE j.datasettkode=ds.datasettkode
+     AND ST_Intersects(geom, ST_GeomFromEWKT(wkt))
+     AND ST_Area(ST_Transform(ST_Intersection(geom, ST_GeomFromEWKT(wkt)),25833))>100 -- 10 m²
+     GROUP BY j.id, j.navn
+    ) x ))) as json
+FROM datasett ds
+WHERE datasettkode = ANY(regexp_split_to_array(kartlag,','));
+$$ LANGUAGE SQL IMMUTABLE;
+
+
+
+curl --header "Content-Type: application/json" \
+  --request POST \
+  --data '{"kartlag":"FYL", "wkt": "POLYGON ((10 60,10 65,20 60,10 60))"}' \
+  https://forvaltningsportalapi.test.artsdatabanken.no/rpc/arealstatistikk_wkt
```

Comment:
Shouldn't we use the new URL? https://okologiskegrunnkartapi.test.artsdatabanken.no/rpc/arealstatistikk_wkt